### PR TITLE
Fix on RVC2 benchmark script

### PR DESCRIPTION
**Pull request by ptoupas**
Fixed RVC2 benchmark script to work with models that have >1 inputs.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+default_language_version:
```
Comment (klemen1999) [inline .pre-commit-config.yaml]: We are not including this part in this (or other libraries) for now as I believe it is a bit setup specific. Can you remove it from the PR and have this change only locally?

**Review by klemen1999 — APPROVED** [review]
LGTM!

File: .pre-commit-config.yaml (hunk 1 above)
Comment (ptoupas) [reply .pre-commit-config.yaml]: Done

**Review by kozlov721 — APPROVED** [review]
LGTM

**Pull request merged**